web/docs/project/env-vars.md

```diff
@@ -92,7 +93,7 @@ These are some general env variables used for various Wasp features:
 { name: "DATABASE_URL", type: "String", isRequired: true, note: "The URL of the PostgreSQL database you want your app to use." },
 { name: "WASP_WEB_CLIENT_URL", type: "URL", isRequired: true, note: "Server uses this value as your client URL in various features e.g. linking to your app in e-mails." },
 { name: "WASP_SERVER_URL", type: "URL", isRequired: true, note: "Server uses this value as your server URL in various features e.g. to redirect users when logging in with OAuth providers like Google or GitHub." },
-{ name: "JWT_SECRET", type: "String", isRequired: true, note: <span>Needed to generate secure tokens. <a href="https://jwtsecret.com/generate" target="_blank" rel="noreferrer">Generate</a> a random string at least 32 characters long.</span> },
+{ name: "JWT_SECRET", type: "String", isRequired: true, note: <><span>Needed to generate secure tokens. </span><SecretGeneratorBlock /></> },
```

Comment:
Two things
- Let's add a line break so it doesn't jump out so much when using it.
- We lost the information about the 32 characters, let's add it back!

```suggestion
{ name: "JWT_SECRET", type: "String", isRequired: true, note: <>A random string of at least 32 characters. Needed to generate secure tokens.<br /><SecretGeneratorBlock /></> },
```